src/utils/dateTimeUtils.js

```diff
@@ -0,0 +1,10 @@
+function getFormatedCurrentTime() {
+  const dateObj = new Date();
+  const hours = dateObj.getHours().toString().padStart(2, 0);
+  const minutes = dateObj.getMinutes().toString().padStart(2, 0);
+  const seconds = dateObj.getSeconds().toString().padStart(2, 0);
```

Comment:
second parameter for padStart should be of type string

Reply:
Done.